body-tracking-samples/sample_unity_bodytracking/Assets/Scripts/TrackerHandler.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.Azure.Kinect.BodyTracking;
+
+
+public class TrackerHandler : MonoBehaviour
+{
+    Dictionary<JointId, int> parentJointMap;
+    public bool drawSkeletons = true;
+
+    // Start is called before the first frame update
+    void Awake()
+    {
+        parentJointMap = new Dictionary<JointId, int>();
+        parentJointMap[JointId.Pelvis] = -1;
```

Comment:
It might be easier to read this map if instead of numbers, we use enum:
```
parentJointMap = new Dictionary<JointId, JointId?>()
{
    { JointId.Pelvis, null },
    { JointId.SpineNavel, JointId.Pelvis },
    ...
};
```